### DISCUSSION TITLE
New command: describe-bindings

### DISCUSSION THREAD

**Discussion by VisenDev** [helix-editor/helix#11708] [Ideas] [answered]
When I started learning how to use the [nyxt browser](https://github.com/atlas-engineer/nyxt), something that was very helpful to me was the `describe-bindings` command. This command shows you [a list of all active keybindings and what they do](https://nyxt-browser.com/documentation#the-nyxt-help-system).

This command was particularly helpful when trying out different sets of default key bindings - because it let me easily figure out the active bindings while inside the application - rather than having to read configuration files or scour for documentation online.

I think a feature like this would be very useful for a modal text editor.

**Comment by kirawi** [comment, marked as answer]
You can do this via the command palette: `S-?`

**Reply by larsks** [reply]
...except you can't, right? The command palette will let you search for commands by name, but there is no facility for looking up the bindings for a particular key.

**Reply by kirawi** [reply]
If you're on master, you type `%b <gk>` for instance.

**Reply by larsks** [reply]
Oh, that's awesome. Thank you!
